.travis.yml

```diff
@@ -31,4 +31,3 @@ after_success:
 # slack notifications configuration
 notifications:
     slack: andela:LTXqtmTsixouhMtsFDOPnaY5
-    
```

Comment:
Deleting this line will not affect the functionality, I suggest you delete it and ensure you leave a newline at the end for the EOL.

Reply:
Done! @Bryan-Cee Thanks